packages/shared/src/atoms/illustrations/BrandLogo.tsx

```diff
@@ -1,12 +1,17 @@
-import type { Breakpoint } from '@mui/material';
+import type { Breakpoint, Theme } from '@mui/material';
+
 import useMediaQuery from '@mui/material/useMediaQuery';
 
-const JumperLogo = ({ theme }) => {
+interface Props {
+  theme: Theme;
+}
+
+const JumperLogo = ({ theme }: Props) => {
   const mainCol =
     theme.palette.mode === 'light'
-      ? theme.palette.accent1.main
-      : theme.palette.accent1Alt.main;
-  const subCol = theme.palette.accent2.main;
+      ? theme?.palette.accent1.main
+      : theme?.palette.accent1Alt.main;
+  const subCol = theme?.palette.accent2.main;
```

Comment:
If it is required in types, why do we need to check for undefined?

Reply:
To be fricking super safe :D 
No, just kidding and got rid of it! Nice find, I happened to use it all across as I started to believe it makes my life with typescipt easier.. but you are completely right: It´s already required and there´s no need for it right here!!